mods/taskbar-clock-customization.wh.cpp

```diff
@@ -76,9 +76,12 @@ Only Windows 10 64-bit and Windows 11 are supported.
 - WebContentsMaxLength: 28
   $name: Web content maximum length
   $description: Longer strings will be truncated with ellipsis
-- WebContentsUpdateInterval: 10
+- WebContentsUpdateInterval: "0.5"
   $name: Web content update interval
   $description: The update interval, in minutes, of the web content
+- WebContentsIterate: true
```

Comment:
Nice idea. With this option, perhaps we'll want to add an optional `WebContentsBlockEnd` option.